candle-examples/examples/mistral/README.md

```diff
@@ -88,3 +88,18 @@ fn quick_sort(arr: &mut [i32]) {
 ``
 226 tokens generated (10.91 token/s)
 ```
+
+## Config
```

Comment:
I don't feel these config details are really needed in the readme, it's more about how to run the examples rather than the technical differences.